sentry-graphql/src/main/java/io/sentry/graphql/SentryInstrumentation.java

```diff
@@ -55,10 +79,107 @@ public SentryInstrumentation() {
   public @NotNull InstrumentationContext<ExecutionResult> beginExecution(
       final @NotNull InstrumentationExecutionParameters parameters) {
     final TracingState tracingState = parameters.getInstrumentationState();
-    tracingState.setTransaction(hub.getSpan());
+    final @NotNull IHub currentHub = Sentry.getCurrentHub();
```

Comment:
Always use `HubAdapter` to make it testable.

Reply:
Hmm I can't. I explicitly need to set one hub for the whole execution as we're using `GraphQLContext` for hub propagation. If I change this to use `HubAdapter` we'd always be using the current threads hub which would lead to bleeding/leeking of breadcrumbs etc. I'll check how to make this testable when I get to automated tests.

Reply:
I meant `Sentry.getCurrentHub()`, if you use the Sentry static class, this won't be testable, you can't mock/fake it.
`Sentry.getCurrentHub()` and `HubAdapter.currentHub` will always return the same thing since HubAdapter delegates to Sentry static class anyway.

Reply:
Found a way to test as is, so leaving it.